editor/test/integration/test_util.clj

```diff
@@ -166,7 +171,7 @@
   resource/Resource
   (children [this] children)
   (ext [this] (FilenameUtils/getExtension (.getPath file)))
-  (resource-type [this] (get (g/node-value workspace :resource-types) (resource/ext this)))
+  (resource-type [this] (#'resource/get-resource-type workspace this))
```

Comment:
I really don't want to make `resource/get-resource-type` public. It's intended to be used internally for each `Resource` implementation of their `resource-type` method like this. Might be OK to access like this since this is only about the tests?